Documentation/Signoffs/Motor System.md

```diff
@@ -0,0 +1,354 @@
+ * The conductors being protected are not part of a branch circuit supplying more than one receptacle for cord-and-plug-connected portable loads.  
+ * The ampacity of the conductors does not correspond with the standard ampere rating of a fuse or a circuit breaker without overload trip adjustments above its rating (but that shall be permitted to have other trip or rating adjustments).  
+ * The next higher standard rating selected does not exceed 800 amperes.  
+^ Source: https://up.codes/viewer/illinois/nfpa-70-2020/chapter/2/wiring-and-protection#240.4_(B)  
+
+The standard NEC 705.12(D)(2) Bus or Conductor Ampere Rating states "One hundred twenty-five percent [125%] of the inverter output circuit current shall be used in ampacity calculations for the following: Feeders, Taps, and Busbars."
+
+^ Source: https://up.codes/viewer/delaware/nfpa-70-2014/chapter/7/special-conditions#705.12_(D)_(2)
+
+* **Will follow the NEC 310-16 Table for its wirings**
+
+This table is for "Allowable Ampacities of Insulated Conductors Rated 0 Through 2000 Volts, 60°C Through 90°C (140°F Through 194°F), Not More Than
+Three Current-Carrying Conductors in Raceway, Cable, or Earth (Directly Buried), Based on Ambient Temperature of 30°C (86°F)"
+
+* **Will be powered by a step-down transformer that will convert the wall voltage down to 24VAC.**
+^ As stated below, the driver we chose for the motor can only be ran with 24-80VAC or 30-110VDC. We will use a step-down transformer to accomplish this.
+
+## Schematic
+
+![Schematic](https://github.com/DillonSW/Capstone_Team_5/blob/Team5-signoff-Motor-System/images/DriverAndMotorRevisionFour.jpg)
+
+The schematic above shows the wiring connections between the microcontroller/microcomputer, driver, and motor. The encoder is a part of the driver and has its own output ports on the driver, however it is labeled as separate in the datasheets. The resistor options for connecting the microcontroller/microcomputer with the driver are: 0 for 5V, 1kΩ for 12V, or 2kΩ for 24V. Because there are 5 resistor connections, 10, 2kΩ resistors would suffice for covering all of them, in which we can use 2, 2kΩ resistors in parallel to create a 1kΩ connection.
+
+Also to note, the HSS86 driver can take between 20-80 AC, or 30-110 DC voltage. Because the wall outlet's nominal voltage is between 110-120 AC voltage, we must step it down in order to safely power the driver.
+^ Source: https://www.jbcnc.se/images/datasheets/HSS86.pdf
+^ Source: https://conquerallelectrical.ca/us-electrical-outlet-voltage/?utm_source=rss&utm_medium=rss&utm_campaign=us-electrical-outlet-voltage
+
+To adhere to the NEC 310-16 table, we are going to implement the following:
+
+* Because the wall outlet will be outputting 120VAC, the equation for current with respect to power consumption is $I = P/V$. The peak current from the driver is 8 Amps, meaning the primary transformer wires will draw $P = 120(8) = 960 W$.
```

Comment:
This is insufficient. It is naive electrically speaking. Analyze the entire power circuit otherwise you are guessing as to whether or not the wires are sized appropriately or not.

Reply:
Let's assume you're right about the primary side current.What happens to the current on the secondary side of the transformer? 

$P_1=P_2$ $P_1=960w$ $P_2 = 960 = 24*I$ 

Now what is $I$ 

Definitely not 8.